### DISCUSSION TITLE
asgiref not installed with fastapi latest version?

### DISCUSSION THREAD

**Discussion by lathkar** [fastapi/fastapi#6262] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
#main.py
from fastapi import FastAPI
app = FastAPI()

@app.get("/")
async def index():
    return {"message": "Hello World"}
```

### Description

A few months ago, I had installed version 0.68 of FastAPI. After installation I had checked the list of packages by running pip freeze where it showed asgiref 3.4.1 version as installed. Now when I install the latest version 0.79, I don't find it in the list. I wonder if it is no more required. (The hello world app runs fine though)

### Operating System

Windows

### FastAPI Version

3.10 '0.79.0'

### Python Version

3.10.1

**Comment by JarroVGIT** [comment, marked as answer]
It is no longer required.

**Comment by lathkar** [comment]
Thanks for your reply
